core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/ReflectiveClassBuildItem.java

```diff
@@ -118,8 +100,9 @@ public boolean isConstructors() {
         return constructors;
     }
 
+    @Deprecated
```

Comment:
We should add a note why it's deprecated and whether we're going to remove it in a future version of quarkus...

The same applies to `finalFieldsWritable()`.

Reply:
We could mention that `finalIsWritable` and `allowUnsafeAccess` are deprecated since GraalVM 21.2.0 and no more used. See the note, I quote: "Parameters {@code finalIsWritable} and {@code allowUnsafeAccess} no longer serve a purpose." Source https://github.com/oracle/graal/blob/vm-21.2.0/sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/hosted/RuntimeReflection.java#L107-L116

Reply:
Warning in 21.1, it is still used as you can see here https://github.com/oracle/graal/blob/vm-21.1.0/sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/hosted/RuntimeReflection.java#L121-L123

Reply:
Done